emsdk.py

```diff
@@ -103,7 +103,10 @@ def errlog(msg):
     BASH = True
 
 if WINDOWS:
```

Comment:
How about `if WINDOWS and not MSYS:`

Also, maybe this can be split into its own change as it a seems like nice separable cleanup.

Reply:
Yeah. That sounds good. This can be a follow up.